packages/stark-ui/src/modules/table/directives/table-row-content.directive.ts

```diff
@@ -0,0 +1,20 @@
+import { Directive, TemplateRef } from "@angular/core";
+
+/**
+ * This directive can be used inside <stark-table> if you want to customise the render of the table's cells
```

Comment:
Could you please add some details about the variables the developer can define ? 
Like: 
* `let rowData = rowData`
* `let cellRawValue = rawValue`
* `let cellDisplayedValue = displayedValue`